spec.bs

```diff
@@ -2266,6 +2267,17 @@ The network revocation mechanism requires the following monkeypatches to the [[F
   (<a href="https://github.com/WICG/fenced-frame/issues/191">WICG/fenced-frame#191</a>)
 </div>
 
+<div algorithm=obtain-a-connection-patch>
+  Modify the [=obtain a connection=] algorithm, insert these steps before step 1:
+
+  1. Let |nonce| be <var ignore>key</var>'s [=network partition key/nonce=].
+
+  1. If the user agent's [=network revocation exemption map=][|nonce|] [=map/exists=], but [=network
+     revocation exemption map=][|nonce|][<var ignore>yrl</var>] does not [=map/exists=], then:
+
+    1. If the user agent's [=network revocation nonce set=] [=set/contains=] |nonce|, then return.
```

Comment:
```suggestion
    1. If the user agent's [=network revocation nonce set=] [=set/contains=] |nonce|, then return
       failure.
```
[The caller](https://fetch.spec.whatwg.org/#ref-for-concept-connection-obtain%E2%91%A0) is expecting some return value, either a connection or failure.